server/website/website/db/base/parser.py

```diff
@@ -227,7 +227,7 @@ def convert_dbms_metrics(self, metrics, observation_time, target_objective):
 
         for target_name, target_instance in target_list.items():
             metric_data[target_name] = target_instance.compute(
-                base_metric_data, observation_time)
+                metrics, observation_time)
```

Comment:
In order to use the wait class name in target objectives, I use the metrics before conversion to calculate the target objectives.